src/app/login/components/feedback-dialog/feedback-dialog.component.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { DebugElement } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { HttpResponse } from '@angular/common/http';
+import { MatDialog, MatDialogModule } from '@angular/material/dialog';
+
+import { of } from 'rxjs';
+import { MockComponent } from 'ng-mocks';
+import Spy = jasmine.Spy;
+
+import { Feedback, FeedbackContext } from '../../models/feedback.model';
+import { FeedbackDialogComponent } from './feedback-dialog.component';
+import { FeedbackFormComponent } from '../feedback-form/feedback-form.component';
+import { FeedbackService } from '../../services/feedback.service';
+import { FeedbackSuccessDialogComponent } from '../feedback-success-dialog/feedback-success-dialog.component';
+import { ErrorDialogComponent } from '../error-dialog/error-dialog.component';
+
+describe('FeedbackDialogComponent', () => {
+  let fixture: ComponentFixture<FeedbackDialogComponent>;
+  let debugElement: DebugElement;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        FeedbackDialogComponent,
+        MockComponent(FeedbackFormComponent)
+      ],
+      imports: [MatDialogModule, HttpClientTestingModule],
+    }).compileComponents();
+
+
+    fixture = TestBed.createComponent(FeedbackDialogComponent);
+    debugElement = fixture.debugElement;
+
+  }));
+
+  it('"feedback" event should trigger handleSubmit and open FeedbackSuccessDialogComponent', () => {
+    const feedback: Feedback = {subject: 'testSubject', body: 'testBody', email: 'test@email.com', context: FeedbackContext.BUG};
+    const feedbackFormComponent = fixture.debugElement.query(By.directive(FeedbackFormComponent));
+
+    const feedbackService = TestBed.inject(FeedbackService);
```

Comment:
wouldn't it be better to move this and the line below into the `beforeEach({...})` section? Right now we just repeat these statements in every test